drivers/bridge/bridge_test.go

```diff
@@ -72,6 +72,71 @@ func TestCreateFail(t *testing.T) {
 	}
 }
 
+func TestCreateMultipleNetworks(t *testing.T) {
+	defer netutils.SetupTestNetNS(t)()
+	d := newDriver()
+	dd, _ := d.(*driver)
+
+	config1 := &networkConfiguration{BridgeName: "net_test_1", AllowNonDefaultBridge: true}
```

Comment:
No, otherwise how do you explain the tests passed ? :-)
The net isolation rules are installed in the FORWARD chain